src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,19 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (comments.length !== 0
+  ? (
```

Comment:
Use 'Array.isArray(comments)' instead of 'comments.length !== 0'. This will ensure that comments is an array before checking its length.